cuegui/cuegui/FrameMonitor.py

```diff
@@ -362,18 +361,20 @@ def _filterLayersHandle(self, action):
         Tells the FrameMonitorTree widget what layers to filter by.
         @param action: Defines the menu item selected
         @type  action: QAction"""
+        layers = self.frameMonitorTree.frameSearch.options.get('layer', [])
         if action.text() == "Clear":
             for item in self._filterLayersButton.menu().actions():
                 if item.isChecked():
-                    self.frameMonitorTree.frameSearch.layers.remove("%s" % item.text())
+                    layers.remove("%s" % item.text())
```

Comment:
I think this can just be `layers.remove(item.text())`? Or maybe `layers.remove(str(item.text()))`

Reply:
Same in a few other spots in this PR.